src/graphql/data/util.ts

```diff
@@ -66,7 +66,7 @@ export const CHAIN_NAME_TO_CHAIN_ID: { [key: string]: SupportedChainId } = {
   OPTIMISM: SupportedChainId.OPTIMISM,
 }
 
-export const BACKEND_CHAIN_NAMES: Chain[] = ['ARBITRUM', 'CELO', 'ETHEREUM', 'OPTIMISM', 'POLYGON']
+export const BACKEND_CHAIN_NAMES: Chain[] = ['ETHEREUM', 'POLYGON', 'OPTIMISM', 'ARBITRUM', 'CELO']
```

Comment:
back out this change - it looks unrelated.

Reply:
This does affect the order in which the ui displays them. I updated it so the explore network filter lists chains in the same order as the global network switcher. Since this doesn't affect the data components but makes UI better I think this is a relevant change. Ultimately a better solution could be to update chainInfo to include a backend chain name field instead of having this list, perhaps in sep PR

Reply:
ah yes I had a sep ticket for this network ordering in the UI